src/index.css

```diff
@@ -46,11 +46,56 @@ body {
     width: 385px;
     height: 330px; }
 
-#title-text {
+#not-found-page {
```

Comment:
The .css files should not be checked in...these are generated by the .scss files.